src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +97,24 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String gender} into an {@code Gender}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
```

Comment:
I think it's better to explain in the header comment what is the input argument `isNaAllowed`